presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -972,6 +981,16 @@ private String formatDomain(Domain domain)
 
             return "[" + Joiner.on(", ").join(parts.build()) + "]";
         }
+
+        private String formatCost(PlanNode node)
+        {
+            PlanNodeCost cost = costs.getOrDefault(node, UNKNOWN_COST);
+            Estimate outputRowCount = cost.getOutputRowCount();
+            Estimate outputSizeInBytes = cost.getOutputSizeInBytes();
+            return String.format("{cost: %s/%s}",
```

Comment:
It's fine to me to have some virtual unit of cost even if it's really only number of rows. Note however that such cost will most likely be used as a number that can be use to relatively compare two plans. It seems intuitive in that form.
However it is getting hard to use for user when there are two of such numbers, as comparing plans becomes non trivial and without knowing what do those numbers represent - virtually impossible.

This is why I would suggest to keep only one number and call it cost, or use two numbers and describe their meaning (eg. using `{rows: %s, bytes: %s}`.